eth/constants.py

```diff
@@ -79,6 +79,7 @@
 
 GAS_MEMORY_QUADRATIC_DENOMINATOR = 512
 
+MAX_REFUND_QUOTIENT = 2
```

Comment:
It's a little surprising to me, for this to be in the global constants file, since it's changing around from fork to fork.

Reply:
This is the original value of the MAX_REFUND_QUOTIENT that had been sprinkled around (as the `2` in `gas_used // 2`), but AIUI not formally named until EIP-3529. I was sort of on the fence about adding it as a constant, but thought it might be easier down the road to do a codebase-wide search for it if it changes again later. Do you think it makes more sense to just keep it as `2`?

Reply:
No, I like having it as a constant, just maybe in frontier fork constants?